### DISCUSSION TITLE
Energy Dashboard & Single Entity (Elektrische Wärmepumpenleistung)

### DISCUSSION THREAD

**Discussion by bufu2012** [LarsK1/hass_solvis_control#72] [General]
Hey Lars! I am just using the Add-On - works great!

Currently I am intgrating it into other Add-Ons, e.g. the Energy-Dashboard. Unfortunately I cannot add any of the Solvis Entities as an energy consumer. This is the appropriate FAQ for [missing entities](https://www.home-assistant.io/docs/energy/faq/#troubleshooting-missing-entities).

What do you think about editing it?

**Comment by LarsK1** [comment]
Hi @bufu2012,
sorry for the late reply. That's true, I intentionally not assigned these units until now, since (as far as I know) no data is available, that scales up over time like a gas usage....

Or did I miss anything?

Thanks for your nice writing!
Lars

**Reply by bufu2012** [reply]
I think we are not on the same side
I mean within the energy dashboard the part of the consumption of a single device. It is possible to add e.g. your whole home consumption and then add particular devices - a fridge, a server or a heatpump. 

Therefore it is necessary to provide that device with an appropriate entity. 
![IMG_6116](https://github.com/user-attachments/assets/51abe102-b401-4c80-9f07-801b8aa9d8f6)
![IMG_6117](https://github.com/user-attachments/assets/b4e2e95b-f43d-4afd-aa84-938b6d9293fe)


So in fact I am talking about the Solvis MIA energy consumption (not some oil or gas). 
Can you follow my thoughts?

**Comment by fkhr79** [comment]
@bufu2012 
In the current dev version 2.0.0-alpha.8 (download via hacs) is a power entity "Elektrische Wärmepumpenleistung". Unit is kW. This should work with the energy dashboard. Will - afaik - also be part of the next release.
Mind: the naming (and other things) might yet change due to alpha-state. But: I'm running this version and it's quite stable!

**Reply by bufu2012** [reply]
Thanks for the remark. 
I am also using the alpha.8 - also stable. For the energy dashboard it is necessary, to use the attributes I wrote within the initial post. E.g. the device_class differs from it. 

Anyway, my solution is a helper by using ‚Integral -> links Riemannsche Summe‘ for the named entity. This calculates the necessary data
